CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 21.57.1
```

Comment:
I'm late to this party, but any change shouldn't also version the gem, we do those separately, see https://github.com/alphagov/govuk_publishing_components/blob/master/docs/publishing-to-rubygems.md